pkg/apis/serving/v1alpha1/revision_lifecycle_test.go

```diff
@@ -56,6 +56,17 @@ func TestRevisionDuckTypes(t *testing.T) {
 	}
 }
 
+func TestRevisionGetConditionSet(t *testing.T) {
+	r := &Revision{}
+
+	actual := r.GetConditionSet().GetTopLevelConditionType()
```

Comment:
the actual/expected vars are possible overkill to just inlining it all in the if statement

Reply:
Simplified it